ui/src/assets/i18n/en.json

```diff
@@ -488,7 +488,8 @@
   "platform.name.iOS": "iOS",
   "browser.name.GOOGLECHROME": "Google Chrome",
   "browser.name.CHROME": "Chrome",
-  "browser.name.FIREFOX": "Mozilla Firefox",
+  "browser.name.MOZILLAFIREFOX": "Mozilla Firefox",
+  "browser.name.FIREFOX" : "Mozilla Firefox",
```

Comment:
@nixonakash01 why do we need duplicate values?

Reply:
some places use only FIREFOX, and some only use MOZILLAFIREFOX. till now, in plans page the text was browser.name.MOZILLAFIREFOX